pkg/reconciler/deployment.go

```diff
@@ -132,5 +132,6 @@ func syncImage(expected corev1.PodSpec, now corev1.PodSpec) {
 		if nc.Image != ec.Image {
 			now.Containers[n].Image = ec.Image
 		}
+		now.Containers[n].Env = ec.Env
```

Comment:
what does this line do?

Reply:
This makes sure that the container has the env variables